wpilibc/src/main/native/include/frc/trajectory/TrajectoryGenerator.h

```diff
@@ -0,0 +1,163 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2019 FIRST. All Rights Reserved.                             */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+#pragma once
+
+#include <memory>
+#include <utility>
+#include <vector>
+
+#include "constraints/TrajectoryConstraint.h"
```

Comment:
Should be "frc/trajectory/constraints/TrajectoryConstraint.h".

Reply:
Resolved in 76d1242